docs/package.json

```diff
@@ -24,7 +24,7 @@
     "@material-ui/docs": "^4.0.0-beta.3",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.56",
-    "@material-ui/monorepo": "https://github.com/mui-org/material-ui.git#master",
+    "@material-ui/monorepo": "https://github.com/m4theushw/material-ui.git#generate-docs",
```

Comment:
To bring back into the master branch

Reply:
Waiting for https://github.com/mui-org/material-ui/pull/26477 to be merged.

Reply:
Merged.